cmd/gpu_plugin/README.md

```diff
@@ -16,7 +16,7 @@ Table of Contents
 ## Introduction
 
 Intel GPU plugin facilitates Kubernetes workload offloading by providing access to
-Intel discrete (Xe) and integrated GPU HW device files.
+discrete (including Intel® Data Center GPU Flex Series) and integrated Intel GPU device files.
```

Comment:
Maybe add also "... supported by the host kernel."?

Reply:
In that case, I'd say: .. integrated Intel GPU devices supported by the host kernel." ? Without the "files".

Reply:
Yes, that's better!

Note: this PR is fine as-is, I can add that kernel note later, in my prerequisites PR.